tensorbay/dataset/segment.py

```diff
@@ -29,6 +30,7 @@
 
 _S = TypeVar("_S", bound="Segment")
 _FS = TypeVar("_FS", bound="FusionSegment")
+_MESSAGES = "The segment name is needed"
```

Comment:
display the version info about when this behavior is deprecated and when it will be removed